### PR TITLE
[HYDRATOR-1371] - Fix Runtime arguments aren"t ordered

### PR DESCRIPTION
**Pull request by sushil-car**
https://issues.cask.co/browse/HYDRATOR-1371

### CODE REVIEW

File: cdap-ui/app/directives/key-value-widget/key-value-widget.html [hunk 1]
```diff
@@ -24,7 +24,7 @@
       <a href="" class="btn btn-danger" ng-click="KeyValue.removeProperty(property)">
         <span class="fa fa-fw fa-trash"> </span>
       </a>
-      <a class="btn btn-info btn-transparent" ng-click="KeyValue.addProperty()" ng-if="$last">
+      <a class="btn btn-info btn-transparent" ng-click="KeyValue.addProperty()" ng-if="$first">
```
Comment (ajainarayanan) [inline cdap-ui/app/directives/key-value-widget/key-value-widget.html]: @sushiladokar why is this modified from `$last` to `$first`?

**Comment by ajainarayanan** [comment]
@sushiladokar Have made the following changes,
- Reverted `$first` to `$last`. Not sure why this was needed (couldn't find in the JIRA as well)
- Modified sorting to happen only on initialize and not everytime a user adds a run time argument

Please test the changes in the UI and see if it exactly matches the JIRA requirements.
  The changes that was made initially actually made using runtime arguments harder.

**Comment by elfenheart** [comment]
👍  LGTM
